src/parser/parser_support.c

```diff
@@ -9699,6 +9729,11 @@ pt_has_non_groupby_column_node (PARSER_CONTEXT * parser, PT_NODE * node, void *a
       return node;
     }
 
+  if (node->info.name.correlation_level != 0)
```

Comment:
it looks better "node->info.name.correlation_level > 0" for readability.

Reply:
Okay. I fixed it.